modules/observe/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -25,24 +25,26 @@ Currently, the {product} Operator does not support creating a `ServiceMonitor` c
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: <custom_resource_name> # <1>
-  namespace: {my-product-namespace} # <2>
+  name: <service-monitor-name> <1>
```

Comment:
Since we're filling in the rest of the values via the attributes, we can fill this value in directly as well with `developer-hub-service-monitor` for easy copying and pasting